.eslintrc

```diff
@@ -516,6 +516,39 @@
         ]
       }
     },
+    {
+      "files": [
+        "packages/**/*.js"
+      ],
+      "excludedFiles": [
+        ".storybook/preview.js",
+        "__mocks__/**/*.js",
+        "packages/activation-notice/**/*.js",
+        "packages/react/**/*.js",
+        "packages/dashboard/src/karma/fixture.js",
+        "packages/story-editor/src/components/canvas/test/useCanvasKeys.js",
```

Comment:
Why is this added?

That file should just do `import { useRef } from '@web-sories-wp/react';`